meinberlin/react/contrib/forms/Input.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export const Input = ({ label, id, before, children, className, ...rest }) => (
+  <div className="form-group">
+    <label htmlFor={id} className="form-label">
+      {label}
+    </label>
+    <div className="input">
+      {before &&
+        <div className="input__before">{before}</div>}
+      <input
+        className={'form-control input__element ' + (before && 'input__element--before ') + className + (children && ' input__element--after')}
```

Comment:
the `className` prop is always `undefined` (at least in this PR) and also ends up as `undefined` in the resulting html, is this intended (e.g. for use in other components?).

Reply:
This was more in case it might come in handy in the future. I haven't needed it here yet, that's true. I was going to use template syntax here first, and then I could have included a check, but apparently eslint doesn't allow that. That's why I left out the check, because I didn't want this line to grow.
If you use something like [classnames](https://www.npmjs.com/package/classnames) or already have similar utility, I will be happy to convert it.

Reply:
template syntax is not allowed because it kept breaking django translations /gettext for us. Might be fixed once the upgrade to django 4.2 is done. I don't think we have anything else in place but @philli-m might know better. If there's no negative side-effect of having `undefined` as  class I'm also fine with leaving it like this

Reply:
I guess we use terney operators sometimes to avoid undefined or maybe write a function to keep the render cleaner